mantid-developer-linux.yml

```diff
@@ -39,6 +40,7 @@ dependencies:
   - scipy>=1.6.2
   - setuptools=48.0.0 # Pinned purposefully due to incompatibility with later versions
   - sphinx<=3.5.4 # Pinned at 3.5.4 and below due to styling changes in 4.0, and 4.3 has a bug fix which imports a lot of matplotlib code (probably correctly, based on what we have in the api/python/mantid/plots/index.html file) and fails the doctests.
+  - sphinx_bootstrap_theme<=0.7.1
```

Comment:
I notice in pip we had `>=0.7.1` but here we have `<=`. Was that intentional?

Reply:
I changed it intentionally to match how we are handling sphinx. We probably don't want the theme to change underneath us.